validator/__init__.py

```diff
@@ -605,6 +605,33 @@ def __call__(self, container):
 
         return (len(errors) == 0, errors)
 
+
+class Email(Validator):
+    """
+    Email verifies if the string given is structured as a valid 
+    email address.
+
+    This does not protect against every possible combination of 
+    email addresses but does a good job to shield against invalid ones.
+
+    # Example
+        dictionary = {
+            "foo": "joe@google.com",
+        }
+
+        validation = {
+            "foo": [Email()],
+        }
+    """
+
+    def __init__(self):
+        self.err_message = "must be a valid email"
+        self.not_message = "must not be a valid email"
+
+    def __call__(self, email):
+        return re.compile(r"^[^.].+@([?)[a-zA-Z0-9-.])+.([a-zA-Z]{2,3}|[0-9]{1,3})(]?)$").match(email)
```

Comment:
All of this looks great, though for consistency's sake I'd prefer this to return a boolean.